state-chain/pallets/cf-cfe-interface/src/lib.rs

```diff
@@ -55,9 +74,14 @@ pub mod pallet {
 	#[pallet::unbounded]
 	pub type CfeEvents<T: Config> = StorageValue<_, Vec<CfeEvent<T>>, ValueQuery>;
 
+	#[pallet::storage]
+	pub type CfeEventsVersion<T: Config> = StorageValue<_, u32, ValueQuery>;
+
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		fn on_initialize(_block_number: BlockNumberFor<T>) -> frame_support::weights::Weight {
+			CfeEventsVersion::<T>::put(CFE_EVENTS_VERSION);
```

Comment:
What's the intention here? This isn't required. Currently what this is doing is setting the CfeEventsVersion to the same thing every block

Reply:
What do you have in mind instead? If you are thinking of using the const value directly, I don't think that would work as it wouldnt' have the correct behaviour around runtime upgrades (historical data wouldn't be accurate). I mentioned that in the linear issue [(PRO-1128)](https://linear.app/chainflip/issue/PRO-1128/store-cfe-events-version-inside-cfe-interface-pallet).

Reply:
You would update it as part of a runtime upgrade and not every block